aws.go

```diff
@@ -119,3 +128,65 @@ func assumeDestinationRole(acfg AwsConfig, creds *credentials.Credentials) (*cre
 
 	return mCreds, *res.Credentials.Expiration, nil
 }
+
+func splitSamlProviderArns(arns string) (*SamlProviderArns, error) {
```

Comment:
Not a big deal, but nitpicking: no real reason for this to return a pointer to this struct; should just return the struct value.

Reply:
(I know that the other code in this project does not follow this guideline well, but that's something I'm hoping to correct...)